modules/hitl/src/views/full/user/index.jsx

```diff
@@ -27,8 +27,14 @@ export default class User extends React.Component {
     let dateFormatted = moment(this.props.session.last_event_on).fromNow()
     dateFormatted = dateFormatted.replace('minutes', 'mins').replace('seconds', 'secs')
 
-    const userAttributes = this.props.session.attributes && JSON.parse(this.props.session.attributes)
-    const textPrefix = this.props.session.direction === 'in' ? 'User: ' : 'Bot: '
+    const userAttributes = this.props.session.attributes && this.props.session.attributes
+
+    const textPrefix =
+      this.props.session.direction === 'in'
+        ? 'User: '
+        : this.props.session.direction === 'out' && this.props.session.source === 'agent'
+        ? 'Agent: '
+        : 'Bot: '
```

Comment:
Please avoid more than one ? : in a single line. Can you split it in two different parts ? It's not clear what it does

Reply:
@allardy Please review #38ee77a